File: python/1074.py
n = int(input())

def oddOrEven(number):
    if number % 2 == 0:
        return 'EVEN'
    elif number % 2 == 1:
        return 'ODD'
    
for i in range(n):
    number = int(input())
    if number > 0:
        print(oddOrEven(number),'POSITIVE')
    elif number < 0:
        print(oddOrEven(number),'NEGATIVE')
    else:
        print('NULL')
        

File: lua/1074.lua
local n = io.read('n')

for i = 1, n, 1 do
    local x = io.read('n')
    local s
    if x == 0 then
        s = 'NULL'
    elseif x % 2 == 0 then
        s = 'EVEN '
    else
        s = 'ODD '
    end

    if x < 0 then
        s = s..'NEGATIVE'
    elseif x > 0 then
        s = s..'POSITIVE'
    end

    print(s)
end